build.gradle.kts

```
@@ -166,7 +166,7 @@ subprojects {
   tasks.withType<KotlinCompile> {
     kotlinOptions {
       if (project.hasProperty("POM_ARTIFACT_ID")) {
-        freeCompilerArgs = listOf("-module-name", project.property("POM_ARTIFACT_ID") as String)
+        kotlinOptions.moduleName = project.property("POM_ARTIFACT_ID") as String
```

This seems redundant, isn't it already in a kotlinOptions block?

```suggestion
        moduleName = project.property("POM_ARTIFACT_ID") as String
```